addons/stock/models/stock_picking.py

```diff
@@ -1050,9 +1055,9 @@ def _pre_action_done_hook(self):
                 return pickings_to_immediate._action_generate_immediate_wizard(show_transfers=self._should_show_transfers())
 
         if not self.env.context.get('skip_backorder'):
-            pickings_to_backorder = self._check_backorder()
-            if pickings_to_backorder:
-                return pickings_to_backorder._action_generate_backorder_wizard(show_transfers=self._should_show_transfers())
+            pickings_backorder_to_confirm = self._check_backorder()
+            if pickings_backorder_to_confirm:
+                return pickings_backorder_to_confirm._action_generate_backorder_wizard(show_transfers=self._should_show_transfers())
```

Comment:
Sorry to notice this only now, but what this change ? The old name is still correct I think